### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by sudeeptarlekar**
Code Refactor

### CODE REVIEW

File: spec/policies/payments/provider_policy_spec.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Payments::ProviderPolicy, type: :policy do
```
Comment (rohitjoshixyz) [inline spec/policies/payments/provider_policy_spec.rb:5]: This seems unrelated

**Review by rohitjoshixyz — APPROVED** [review]
Check the unrelated file. Thanks for working on this 👍 LGTM 🎉

**Review by apoorv-mishra — APPROVED** [review]
LGTM 👍

File: spec/policies/payments/provider_policy_spec.rb (hunk 1 above)
Comment (sudeeptarlekar) [reply spec/policies/payments/provider_policy_spec.rb:5]: Yeah, spec coverage was less, so thought will add here instead of creating extra PR

Comment (rohitjoshixyz) [reply spec/policies/payments/provider_policy_spec.rb:5]: Okay
